src/main/java/pl/javastart/task/Main.java

```diff
@@ -1,8 +1,30 @@
 package pl.javastart.task;
 
+import java.util.Scanner;
+
 public class Main {
 
     public static void main(String[] args) {
-
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Podaj liczbe");
+        int number = scanner.nextInt();
+        int min = 100;
+        int max = 200;
+        while (number % 3 != 0) {
+            if (number > max) {
+                System.out.println("Liczba jest za duza");
+                System.out.println("Podaj liczbe");
+                number = scanner.nextInt();
```

Comment:
Ten fragment powtarza się w Twoim kodzie aż 4 razy.
Spróbuj to zamienić na pętlę do-while, albo jakąś wersję z break, żeby było to zapisane w kodzie tylko raz